backend/src/main/java/database/EncryptedDatabase.java

```diff
@@ -0,0 +1,17 @@
+package database;
+
+import com.j256.ormlite.field.DatabaseField;
+import com.j256.ormlite.table.DatabaseTable;
+
+@DatabaseTable(tableName = "encrypted")
+public class EncryptedDatabase {
```

Comment:
Du kannst die Klasse in das "models" package verschieben und umbenennen (EncryptedModel). An sich ist das ja keine Datenbank :)